web/packages/design/src/Icon/Icon.story.tsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * Teleport
+ * Copyright (C) 2025 Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { useEffect, useRef, useState } from 'react';
+import { CSSTransition } from 'react-transition-group';
+import styled from 'styled-components';
+
+import { Broadcast } from './Icons/Broadcast';
+
+export default {
+  // Nest stories under Icon/Icon, so that Icon/Icons which lists all icons is the first story.
+  title: 'Design/Icon/Icon',
+};
+
+export const WithRef = () => {
+  const [isShown, setIsShown] = useState(true);
+  const nodeRef = useRef(null);
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setIsShown(value => !value);
+    }, timeoutMs * 3);
```

Comment:
Should this be configurable via prop?

Reply:
If it was a real component then sure, but I don't think it matters that much for the story.